core/src/SYCL/Kokkos_SYCL_Instance.hpp

```diff
@@ -109,7 +109,7 @@ class SYCLInternal {
         assert(m_mem);
         assert(sizeof(T) == m_mem->size());
 
-        if constexpr (sycl::usm::alloc::device == kind)
+        if constexpr (sycl::usm::alloc::device == Kind)
```

Comment:
Where is this change coming from?

Reply:
There was a public member

`static constexpr sycl::usm::alloc kind = Kind;`

which seems to have vanished.

Normally I try and expose all the (non-parameter-pack) template parameters via typedefs and static constexpr constants, as it is much easier when users of the class need to query them.